site/en/blog/expanding-privacy-sandbox-testing-2023/index.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: Expanding Privacy Sandbox for the Web testing into 2023
+title: "Privacy Sandbox for the web: expanding testing into 2023"
```

Comment:
```suggestion
title: "Privacy Sandbox for the web: Expanding testing into 2023"
```

Per the style guide: https://developers.google.com/style/capitalization#capitalization-in-titles-and-headings